content-services/latest/upgrade/index.md

```diff
@@ -37,6 +43,7 @@ The following table shows the upgrade path for major versions:
 | Community Edition 201806 GA | Content Services 6.0.0 |
 | Community Edition 201901 GA | Content Services 6.1.0 |
 | Community Edition 201911 GA | Content Services 6.2.0 |
+| Community Edition 7.0 | Content Services 7.0.0 |
```

Comment:
Should these both be either .0.0 or just .0 ?
Given it appears we are trying to make them look parallel ?

Reply:
They are both correct. If you want to align them, I suggest that you drop the last zero from the right column It should say 6.0, 6.1, 6.2, 7.0 because the paths are not to 6.2.0 but also to 6.2.2, etc.